packages/documentation/src/stories/getting-started/getting-started.docs.mdx

```diff
@@ -0,0 +1,135 @@
+import { Meta, Source } from '@storybook/blocks';
+import * as GettingStartedStories from './getting-started.stories';
+
+<Meta of={GettingStartedStories} />
+
+# Getting started
+
+Welcome to the Swiss post design system world!
+
+## Which package do I need?
+
+The design-system is formed with several packages and different compatibilities with your favorite framework.
+
+<div role="alert" className="alert alert-info">
+  All components require at least the <a href="/?path=/docs/e53e2de8-0bbf-4f70-babc-074c5466f700--docs"><span aria-hidden="true">📦 </span>Styles package</a>.
+</div>
+
+<div className="table-responsive">
+  <table className="table">
+    <caption className="visually-hidden">Compatibilities between components and frameworks and which package is needed</caption>
+    <colgroup>
+      <col style={{width: '25%'}}/>
+      <col style={{width: '25%'}}/>
+      <col style={{width: '25%'}}/>
+      <col style={{width: '25%'}} />
+    </colgroup>
+    <thead>
+      <tr>
+        <th scope="col">Component</th>
+        <th scope="col">JavaScript</th>
+        <th scope="col">Angular</th>
+        <th scope="col">React</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>Internet-header</td>
+        <td>
+          <a href="/?path=/docs/cfa21cbc-5f8d-4a42-8165-7f6ed20bf133--docs"><span aria-label="Package:">📦 </span>Internet-header</a>
+        </td>
+        <td>
+          <a href="/?path=/docs/cfa21cbc-5f8d-4a42-8165-7f6ed20bf133--docs"><span aria-label="Package:">📦 </span>Internet-header</a> with <a href="/?path=/docs/8426deb2-3d6d-4fcc-90e6-1ca9737b2ed0--docs">CUSTOM_ELEMENTS_SCHEMA</a>
+        </td>
+        <td >TBD</td>
+      </tr>
+      <tr>
+        <td>Intranet-header</td>
+        <td>N/A</td>
+        <td>
+          <a href="/?path=/docs/7068be42-1a20-4a34-9e79-7fb7f9f2940f--docs"><span aria-label="Package:">📦 </span>Intranet-header</a>
+        </td>
+        <td >N/A</td>
+      </tr>
+      <tr>
+        <td>
+          Web Components:
+            <ul>
+              <li><a href="/?path=/docs/4d1b4185-e04d-494a-ab38-2b56c1778b0b--docs">Accordion</a></li>
+              <li><a href="/?path=/docs/105e67d8-31e9-4d0b-87ff-685aba31fd4c--docs">Alert</a></li>
+              <li><a href="/?path=/docs/6a91848c-16ec-4a23-bc45-51c797b5b2c3--docs">Collapsible</a></li>
+              <li><a href="/?path=/docs/40ed323b-9c1a-42ab-91ed-15f97f214608--docs">Icons</a></li>
+              <li><a href="/?path=/docs/9a636763-de2d-4f72-bc81-98daf10871f7--docs">Popover</a></li>
+              <li><a href="/?path=/docs/bb1291ca-4dbb-450c-a15f-596836d9f39e--docs">Tabs</a></li>
+              <li><a href="/?path=/docs/cd684d90-e7a7-41a9-8923-b1b72ad9b384--docs">Tooltip</a></li>
```

Comment:
Here is the issue to automate the creation of this list: https://github.com/swisspost/design-system/issues/2728